### DISCUSSION TITLE
公网非80，443标准端口服务的图片预览(Image)、download的url访问没带上端口，导致无法访问和无法下载

### DISCUSSION THREAD

**Discussion by Throokie** [AlistGo/alist#4013] [General]
![(TSF984 U_)U `JHY2R7YWQ](https://user-images.githubusercontent.com/58783886/229185179-4cdce1bc-dddd-47a9-a28a-3e658765739c.png)

![image](https://user-images.githubusercontent.com/58783886/229183481-d191972b-51f1-42b5-98ce-ca6b20ad9c7a.png)

如图所示。加上端口就正常了
![image](https://user-images.githubusercontent.com/58783886/229183965-8e16f16e-89b8-4f63-8ce4-e8e0e3e232ae.png)

应该可以像博客那样添加一个客户自己定义的网站地址来解决。这样我就可以往里面加端口了

**Comment by wy580477** [comment]
在config.json文件里面设置site_url了么？

**Reply by yydylab** [reply]
请问site_url怎么填写，比如我的域名是：https://alist.huhu.com:2023

**Reply by yydylab** [reply]
报错日志：INFO[2023-07-31 10:53:19] reading config file: data/config.json
FATA[2023-07-31 10:53:19] load config error: conf.Config.SiteURL: ReadObject: expect : after object field, but found h, error found in #10 byte of ...|site_url"https://ali|..., bigger context ...|{
  "force": false,
  "site_url"https://alist.huhu.com:2023"",
  "cdn": "",
  "jwt_|...

**Reply by yydylab** [reply]
已解決：                       
{
  "force": false,
  "site_url": "http://example.com",
}
是双引号问题，需要把域名写在冒号后面的双引号内才行。

**Comment by Throokie** [comment]
好，问题解决了，蟹蟹&nbsp;


326516678
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月1日(星期六) 上午8:22
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 公网非80，443标准端口服务的图片预览(Image)、download的url访问没带上端口，导致无法访问和无法下载 (Discussion #4013)





 
在config.json文件里面设置site_url了么？
